src/Components/Posts.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import axios from 'axios';
+import UnfoldMoreDoubleIcon from '@mui/icons-material/UnfoldMoreDouble';
+
+export default class Posts extends React.Component {
+    constructor(props) {
+    super(props);
+  
+    // Initializing the state 
+    this.state = {
+        renderType:'posts',
+        items: [],
+        // revData:false
```

Comment:
why setting items as [] in every component constructor , no use case of these , pass from parent using props